### PR TITLE
Align tutorial theming with the mockups

### PR DESCRIPTION
**Pull request by ferndot**
Screenshot_1586462603
Screenshot_1586462605
Screenshot_1586462624

### CODE REVIEW

**Comment by gspencergoog** [comment]
What does it look like?

**Comment by ferndot** [comment]
@gspencergoog I added screenshots

File: lib/src/ui/screens/tutorial/steps/intro.dart [hunk 1]
```diff
@@ -57,29 +58,29 @@ class TutorialIntroStep extends StatelessWidget {
                       icon: FaIcon(
                         FontAwesomeIcons.mapMarked,
                         size: 30,
+                        color: Theme.of(context).scaffoldBackgroundColor,
                       ),
-                      leadingBackgroundColor:
-                          Theme.of(context).accentColor.withOpacity(0.5),
+                      leadingBackgroundColor: Theme.of(context).primaryColor,
```
Comment (gspencergoog) [inline lib/src/ui/screens/tutorial/steps/intro.dart]: I'd use `Theme.of(context).colorScheme.primary` here and elsewhere for consistency with the rest of the code. Color scheme is a more "modern" Flutter usage anyhow.

Comment (ferndot) [reply lib/src/ui/screens/tutorial/steps/intro.dart]: 👍 sounds great! I'll start using that going forward.

Comment (ferndot) [reply lib/src/ui/screens/tutorial/steps/intro.dart]: @gspencergoog fixed up. I also replaced this usage in other widgets (outside the scope of this PR, but I wanted to move quickly)

**Review by gspencergoog — APPROVED** [review]

**Pull request merged**